servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -266,6 +266,23 @@ bool RenderForwardClustered::free(RID p_rid) {
 	return false;
 }
 
+bool RenderForwardClustered::get_material_has_transparency(RID p_material_rid) const {
+	SceneShaderForwardClustered::MaterialData *material = dynamic_cast<SceneShaderForwardClustered::MaterialData *>(RendererRD::MaterialStorage::get_singleton()->material_get_data(p_material_rid, RendererRD::MaterialStorage::SHADER_TYPE_3D));
+	// Can't found the SceneMaterialData corresponding to p_material_rid
+	if (!material)
+		return false;
+
+	bool has_read_screen_alpha = material->shader_data->uses_screen_texture || material->shader_data->uses_depth_texture || material->shader_data->uses_normal_texture;
+	bool has_base_alpha = (material->shader_data->uses_alpha && (!material->shader_data->uses_alpha_clip || material->shader_data->uses_alpha_antialiasing)) || has_read_screen_alpha;
+	bool has_blend_alpha = material->shader_data->uses_blend_alpha;
+	bool has_alpha = has_base_alpha || has_blend_alpha;
+
+	if (has_alpha || has_read_screen_alpha || material->shader_data->depth_draw == SceneShaderForwardClustered::ShaderData::DEPTH_DRAW_DISABLED || material->shader_data->depth_test == SceneShaderForwardClustered::ShaderData::DEPTH_TEST_DISABLED) {
+		return true;
+	}
+	return false;
+}
+
```

Comment:
Based on : https://github.com/godotengine/godot/blob/master/servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp#L3730

I could have used my new method in place of the original code, but I didn't want to add regressing in a bugfix.